src/main/java/no/uib/inf112/core/player/Robot.java

```diff
@@ -107,8 +99,8 @@ public void move(Movement movement) {
      * Move the robot with given delta to new coordinates
      */
     private void move(int deltaX, int deltaY) {
-        this.x = x + deltaX;
-        this.y = y + deltaY;
+        x = x + deltaX;
+        y = y + deltaY;
     }
```

Comment:
not super fan of this change, is it better to do this.x += deltax, or just revert to old notation